wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/en.md

```diff
@@ -258,49 +258,54 @@ This article lists countries that have participated in official [osu!](/wiki/Gam
 
 ### ![][osu!mania] osu!mania World Cup
```

Comment:
This section should probably be considered for splitting up into 4K and 7K sub-sections as the table is *rather* big.

Reply:
maybe the whole article should